### ISSUE TITLE
Scan works, but output unworkable

### ISSUE THREAD

**Issue by lzandman** [dtact/spring4shell-scanner#4]
The scanner seems to work just fine. However, the output this tool produces isn't really suitable for further processing:

1. There's no built-in feature for logging to a file, which seems like it should be one of the primary use-cases.
2. There doesn't seem to be a way to log only the vulnerable stuff. I.e. there's no `--quiet` parameter. It outputs thousands of lines, one for each file scanned. But I think most people are only interested in the actual vulnerable files. Of course you can pipe the output to a filter (i.e. filter on "[!]"), but then there's still the problem mentioned at point 4 below.
3. The `--json` parameter doesn't seem to work. I still get the regular output.
4. The output text contains all kinds of control characters. Even when using the `--disable-colour` parameter. So redirecting the output to a text file results in a non-plain text file.

I wanted to love this scanner, but in its current state i cannot recommend it.

_Note: I tested the Windows version_

**Comment by mikef-nl** [comment]
@lzandman See https://github.com/dtact/spring4shell-scanner/pull/3 for the first item. It will then log only vulnerabilities and end result to a path.

**Comment by lzandman** [comment]
> @lzandman See #3 for the first item. It will then log only vulnerabilities and end result to a path.

Yeah, saw that PR. But didn't have time to review it and create a build. Would be good if @nl5887 included it or made some additions of his own. We are now using a different scanner.

**Comment by nl5887** [comment]
Sorry @lzandman but I've some issues with your approach. You don't have time to review and create build, but you are expecting me to make time for this? We're working on a different version of the scanner, including more enterprise like features. This version was quickly created, just to help people identify vulnerabilities. You're completely free to use this version, or a different scanner.
